test/tla/PickPerf2.tla

```diff
@@ -0,0 +1,40 @@
+----------------------- MODULE PickPerf2 --------------------------------------
+EXTENDS Integers
+
+VARIABLES
+    \* @type: Set(Int);
+    S,
+    \* @type: Set(Int);
+    T,
+    \* @type: Set(Int);
+    R,
+    \* @type: Int;
+    state
+
+Init ==
+    /\ S = { 1 }
+    /\ T = { 4 }
+    /\ R = S \union T
+    /\ state = 0
+
+Next ==
+  /\\/ 1 \in S /\ S' = { 2 } \union S /\ UNCHANGED T
+    \/ 2 \in S /\ S' = { 3 } \union S /\ UNCHANGED T
+    \/ 3 \in S /\ S' = { 1 } \union S /\ UNCHANGED T
+    \/ 4 \in T /\ T' = { 5 } \union T /\ UNCHANGED S
+    \/ 5 \in T /\ T' = { 6 } \union T /\ UNCHANGED S
+    \/ 6 \in T /\ T' = { 1 } \union T /\ UNCHANGED S
+  /\ R' = S' \union T'
+  /\ state' \in 0..4
```

Comment:
Would it make any difference to have it as `state \in 0..3`?

Reply:
Yes, one of the cases below has `state' = 4`. If we write `state' \in 0..3`, this case will be excluded

Reply:
I meant `state` no primed, and looking at it again `state \in 0..2` would yield the same I think.

Reply:
I see your point. Using `state \in 0..2` might work in a similar way but it does have the effect you intend.